babel.config.js

```diff
@@ -19,11 +21,13 @@ const getPresets = () => {
           browsers: ['last 2 versions'],
           node: '8',
         },
+        useBuiltIns: 'usage',
         modules: false,
       }
     case 'cli':
       return {
         targets: { node: '8' },
+        useBuiltIns: 'usage',
```

Comment:
These are only needed as long as we support `Node 8` since it does not support Promise#finally.

Can be removed as soon as we move to Node 10/12 pairings.